app/scripts/modules/core/src/cluster/cluster.service.ts

```diff
@@ -40,7 +42,8 @@ export class ClusterService {
       return serverGroupLoader.getList().then((serverGroups: IServerGroup[]) => {
         serverGroups.forEach(sg => this.addHealthStatusCheck(sg));
         serverGroups.forEach(sg => this.addNameParts(sg));
-        return this.$q.all(serverGroups.map(sg => this.serverGroupTransformer.normalizeServerGroup(sg, application)));
+        return this.$q.all(serverGroups.map(sg => this.serverGroupTransformer.normalizeServerGroup(sg, application)))
+          .then(normalized => normalized.filter(sg => !!sg));
```

Comment:
super nit: `.filter(Boolean)` is a little more idiomatic/less noisy.

Reply:
👍